### ISSUE TITLE
LESS: Minus ('-') is wrongly considered as unknown function

### ISSUE THREAD

**Issue by salixzs** [racodond/sonar-css-plugin#67]
SonarQube server version 6.4.0.25310
CSS / SCSS / Less plugin, version 4.5

I receive this violation:
**Remove this usage of the unknown "-" function.**
It is reported as "_Unknown CSS / Less functions should be removed_" violation.
LESS code piece is actually normally compiling and working code:
```css
.step-marker {
    top: -(@timeline-h-size-step/2)-(@timeline-h-size-line/2);
}
```
(both minus signs are reported as unknown function.)

**Comment by racodond** [comment]
Hi @salixzs,

Thanks a lot for your feedback!
The issue should now be fixed. Feel free to give a try to the following snapshot and provide some feedback: https://github.com/racodond/sonar-css-plugin/releases/tag/%2367

Thanks!

David

**Comment by salixzs** [comment]
Thanks. After using this pre-release these false positives was removed ("fixed"). All other problems reported seems to be staying without change (OK situation).

**Comment by racodond** [comment]
Cool! Thanks for the feedback! I'll proceed with an official release tonight.
